internal/transactions/send-signed.go

```diff
@@ -0,0 +1,60 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019-2021 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package transactions
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-cli/internal/command"
+	"github.com/onflow/flow-cli/pkg/flowcli/services"
+)
+
+type flagsSendSigned struct {
+}
+
+var sendSignedFlags = flagsSendSigned{}
+
+var SendSignedCommand = &command.Command{
+	Cmd: &cobra.Command{
+		Use:     "send-signed <signed transaction filename>",
+		Short:   "Send signed transaction",
+		Args:    cobra.ExactArgs(1),
+		Example: `flow transactions send-signed signed.rlp`,
+	},
+	Flags: &sendSignedFlags,
```

Comment:
Question not specific to this PR: does it work to just omit the `Flags` field when it isn't needed? (That's the same as `Flags: nil`)

Reply:
Will try it out, or else make it work with nil, good idea.

Reply:
Doesn't work out of the box, I can refactor this at a later time